managed/services/telemetry/datasource_qandb_select.go

```diff
@@ -65,7 +65,7 @@ func openQANDBConnection(dsn string, enabled bool) (*sql.DB, error) {
 		return nil, errors.Wrap(err, "Failed to open connection to QAN DB")
 	}
 	if err := db.Ping(); err != nil {
-		return nil, errors.Wrap(err, "Failed to ping QAN DB")
+		l.Warnf("DB is not reachable [%s]", dsn)
```

Comment:
fixes [PMM-10207](https://jira.percona.com/browse/PMM-10207)

Reply:
we don't have ping there

Reply:
so what will happen in such a case?

Reply:
I checked the impl, in case of VM it is a wrapper around HTTP. DB - manages connection pool and has ping.

Reply:
Please print the `err` here into log stream.

Reply:
done